PIL/Image.py

```diff
@@ -1697,7 +1705,7 @@ def resize(self, size, resample=NEAREST):
 
         self.load()
 
-        size = tuple(size)
+        size = (size[0], size[1])  # Truncate input into 2-tuple
```

Comment:
I wasn't sure exactly what was happening here, but mypy was concerned that a parameter annotated as a `Tuple[int, int]` became a less-defined general tuple. This seemed like a more explicit way of saying that size must be a 2-tuple at run-time.

Should functions accepting something which is a size, all be calling _check_size()? Should I amend this/these commits to have _check_size(), add another, or is it good as-is? (and maybe handle general _check_size(size) adding in a separate PR?)